src/igvfd/searches/configs/curated_set.py

```diff
@@ -49,17 +67,40 @@ def curated_set():
             'audit.INTERNAL_ACTION.category': {
                 'title': 'Audit Category: Internal Action'
             },
+            'type': {
+                'title': 'Object Type'
+            },
         },
         'facet_groups': [
             {
-                'title': 'File Set',
+                'title': 'Curated Set Details',
                 'facet_fields': [
-                    'taxa',
                     'file_set_type',
+                    'taxa',
                     'assemblies',
                     'transcriptome_annotations',
                 ],
             },
+            {
+                'title': 'Sample',
+                'facet_fields': [
+                    'donors.taxa',
+                    'samples.classifications',
+                    'samples.sample_terms.term_name',
+                    'samples.targeted_sample_term.term_name',
+                    'samples.disease_terms.term_name',
+                    'samples.modifications.modality',
+                    'samples.treatments.treatment_term_name',
+                    'sequencing_library_types',
```

Comment:
Looks like this is like auxiliary sets — need to add `sequencing_library_types` to the `facets` object.

Reply:
copy pasted error. I will remove